clients/src/main/java/org/apache/kafka/common/security/oauthbearer/internals/OAuthBearerSaslClient.java

```diff
@@ -116,6 +129,20 @@ public boolean hasInitialResponse() {
         }
     }
 
+    private SaslExtensionsCallback parseCustomExtensions() throws SaslException {
```

Comment:
I kind of want to have this in a more general space where every SaslClient will have this code.
An idea could be extend the SaslClient interface and provide this default method, or move it to some utils resource.
I'm not sure if it is worth the effort.

Reply:
This still holds even though it got minimized by Github